tide_core.install

```diff
@@ -372,3 +372,26 @@ function tide_core_update_8014() {
     $module_installer->install(['link_field_autocomplete_filter']);
   }
 }
+
+/**
+ * Config for Seckit module.
+ */
+function tide_core_update_8015() {
+  if (\Drupal::moduleHandler()->moduleExists('seckit')) {
```

Comment:
@MdNadimHossain , you need to enable it if it's not enabled. It's already in composer.json, so we know that it is in the codebase.

Reply:
Hi @kurtfoster , As we discussed, it is added as a dependency in this module so I am only checking if the module exists and then updating the config.